### PR TITLE
difference between producer and presenter

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/1673

### CODE REVIEW

**Comment by scanum** [comment]
@tonysmit Hi Tony, changes look correct. Could you please check and merge? Thanks!

**Comment by LolaJacobsen** [comment]
@LanaChin - could you review these changes and (if correct) please merge this PR.

File: Teams/teams-live-events/what-are-teams-live-events.md [hunk 1]
```diff
@@ -33,6 +33,14 @@ So, let's get started. First, take a look at the following diagram that shows hi
 
 ![Diagram showing key components of live events, scheduling, production, Microsoft Stream platform, certified third-party eCDN providers](../media/teams-live-events.png  "Diagram showing key components of live events, scheduling, production, Microsoft Stream platform, certified third-party eCDN providers")
 
+### Planning live events in Microsoft Teams
+Live events in Teams empowers multiple roles to successfully broadcast and participate in an event, including the following: organizer, producer, presenter, and attendee.
+
+- **Organizer**: The organizer manages the live event end to end. This includes creating the event with appropriate attendee permissions, choosing the production method, inviting event team members and attendees and managing event reports. Event reports will include both recording and attendance reports.  With real-time captions & translation (coming soon), the organizer can also specify whether live event attendees can view captions in the supported languages.
```
Comment (nenonix) [inline Teams/teams-live-events/what-are-teams-live-events.md]: **Event reports include recordings and attendance reports.**

File: Teams/teams-live-events/what-are-teams-live-events.md [hunk 2]
```diff
@@ -33,6 +33,14 @@ So, let's get started. First, take a look at the following diagram that shows hi
...
+
+- **Producer**: As a host of the meeting, producers ensure attendees have a great viewing experience by controlling the media sources that are sent to the event. Additional capabilities include: Start / Stop, Share own video, Share participant video, Share active desktop/window, Select layouts, Chat with other event team members, View live attendee count, Moderate Q & A.
```
Comment (nenonix) [inline Teams/teams-live-events/what-are-teams-live-events.md]: **As the host of a meeting,**

Comment (nenonix) [inline Teams/teams-live-events/what-are-teams-live-events.md]: Suggestion:
**Additional capabilities include: Start / Stop, share own video, share participant video, share active desktop/window, select layouts, chat with other event team members, view live attendee count, and moderate Q & A.**

Comment (VLG17) [reply Teams/teams-live-events/what-are-teams-live-events.md]: @nenonix implemented suggestions

**Comment by LanaChin** [comment]
Awaiting review by PM.

**Comment by scanum** [comment]
@LanaChin Hi Lana, any updates on this? Thanks!

**Comment by LolaJacobsen** [comment]
@LanaChin could you ping PM again please?

**Comment by scanum** [comment]
@SerdarSoysal @LanaChin Hi any updates on this? Thanks!

**Comment by LanaChin** [comment]
@scanum I met with the PM and updated the article to add a link to an existing article that describes each of the roles.  Thanks.

**Pull request merged**